README.md

```diff
@@ -1,43 +1,55 @@
-# Front-end Starting App
+[![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)](https://github.com/bcgov/nr-frontend-starting-app)
 
-This is a starting front-end app that intends to get the team started with code definitions, commits standards and guides related to code-style.
+# Natural Resources Front-End Starting PWA
 
-In case you're looking for the original readme template file, you can see it [here](README_template.md)
+This repository holds a set of policies, standard, guides and pipelines to get
+started with a PWA. Before writing your first line of code, please take a
+moment and check out our [CONTRIBUTING](CONTRIBUTING.md) guide.
```

Comment:
Suggestion: use the meaning of PWA (progressive web application, right?) in the header and again in the paragraph; _then_ we introduce the acronym. For instance:

```
# Natural Resources Front-End Starting Progressive Web Application

[...] to get stared with a progressive web application (PWA). [...]
```

And we can use _PWA_ from then on.

Reply:
Nice. Fixed.